test/transforms.jl

```diff
@@ -145,6 +145,100 @@
     end
   end
 
+  @testset "UniqueCoords" begin
+    X = [i * j for i in 1:2, j in 1:100]
```

Comment:
We tested with 1_000_000 to make sure that the implementation scales. If we remove the Base implementation of uniqueinds for example the algorithm takes forever. Can you please use 1_000_000 again?